server/command_import.go

```diff
@@ -15,13 +16,24 @@ func (p *Plugin) runImportCommand(args []string, extra *model.CommandArgs) (*mod
 	}
 	installationDNS := standardizeName(args[0])
 
-	splitDNS := strings.Split(installationDNS, ".")
+	u, err := url.Parse(installationDNS)
+	if err != nil {
+		return nil, true, errors.Wrap(err, "error parsing url")
+	}
+
+	hostname := u.Hostname()
+
+	if hostname == "" {
```

Comment:
nit: it may be good to include a comment with what we encountered about the hostname being blank if no protocol is included. That could help future engineers if they review the code and are wondering what this does.